src/quantum/azext_quantum/operations/__init__.py

```diff
@@ -0,0 +1,4 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If I recall, the updated guidance is that the "All rights reserved." part of headers should be dropped?

Reply:
this is what the files came with.